README.md

```diff
@@ -1 +1,33 @@
+# What is the Weather? App
+
+A weather application to demonstrate API calls and Manipulation.
+
+## Install (MAC)
+
+* Install Homebrew ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"```
+* Install the most current version of node.js ```brew install node```
+* Download this "weather" project file
+* In terminal, navigate to your weather file.
+* Open example.env and follow the directions there.
+* Run the following commands to install dependencies ``npm install axios`` and ``npm install dotenv``
+* Enjoy!
+
+## Usage
+
+Have your terminal opened to the weather directory.
+
+```zsh
+node app.js {your-city},{your-state}
+```
+
+or
+
+```zsh
+node app.js {zip-code}
+```
+
+## City and State Format
```

Comment:
I would next this under the usage section.

Reply:
Done.